tests/components/fronius/test_config_flow.py

```diff
@@ -145,33 +145,6 @@ async def test_form_cannot_connect(hass: HomeAssistant) -> None:
     assert result2["errors"] == {"base": "cannot_connect"}
 
 
-async def test_form_no_device(hass: HomeAssistant) -> None:
```

Comment:
since we are always a bit picky about entirely testing the config flow, can we add the `StopIteration` to `test_form_cannot_connect` through parametrization?

Reply:
done